src/storybook/__snapshots__/Stories.test.ts.snap

```diff
@@ -2107,7 +2107,7 @@ exports[`Storyshots Attributes / Attributes selected 1`] = `
                 class="MuiInputBase-input-id MuiOutlinedInput-input-id"
                 name="attribute:Date Time Attribute:time"
                 type="time"
-                value="00:15"
+                value="02:15"
```

Comment:
Use the environment variable, if the command does not work on your system - set the env manually or modify the command to be cross-platform.

Reply:
Indeed, it is probably due to some problems with that command on Windows. That was the output after making the command cross-platform. Setting env manually did't work for me.

Reply:
Call `set TZ=UTC` before running `jest`. Removing half of the command is not the same as making it cross-platform 😉

Reply:
I merely preceded the command with `cross-env`. I didn’t modify it in any other way.